tests/core/test_test.py

```diff
@@ -256,13 +256,54 @@ async def test_action_unlikely_intent_correctly_predicted(
     )
 
     result = await rasa.core.test.test(
-        str(file_name), agent, out_directory=str(tmp_path),
+        str(file_name), agent, out_directory=str(tmp_path), fail_on_prediction_errors = True
```

Comment:
@alwx should this be set to `True` in all the tests of this PR using `rasa.core.test.test`?

Reply:
I don't think it should be set to `True` at least anywhere else. Otherwise we can't test the report as it won't be created.

Reply:
For some but not for all — we sometimes also need to test if the generated report is correct, and `fail_on_prediction_errors=True` means no report will be generated.